css-view-transitions-2/Overview.bs

```diff
@@ -264,6 +281,34 @@ Note: this event is fired when [=reveal document|revealing a document=].
 The <dfn attribute for=PageRevealEvent>viewTransition</dfn> [=getter steps=] are to return the
 [=inbound cross-document view-transition=] for [=this's=] [=relevant global object's=] [=associated document=].
 
+## Additions to {{Document}} ## {#additions-to-document-api}
+
+	<xmp class=idl>
+		dictionary StartViewTransitionOptions {
+			UpdateCallback? update = null;
+			sequence<DOMString>? types = null;
+		};
+
+		partial interface Document {
+
+			ViewTransition startViewTransition(optional (UpdateCallback or StartViewTransitionOptions)? callbackOrOptions = null);
+		};
+	</xmp>
+
+### {{Document/startViewTransition(options)}} Method Steps ### {#ViewTransition-start-with-options}
+
+	<div algorithm="start-vt-with-options">
+		The [=method steps=] for <dfn method for=Document>startViewTransition(|callbackOrOptions|)</dfn> are as follows:
+
+		1. If |callbackOrOptions| is an {{UpdateCallback}}, then run the [=method steps=] for {{Document/startViewTransition(updateCallback)}} given |callbackOrOptions| and return the result.
+
+		1. Let |viewTransition| be the [=method steps=] for {{Document/startViewTransition(updateCallback)}} given |callbackOrOptions|'s {{StartViewTransitionOptions/update}}.
+
+		1. Set |transition|'s [=ViewTransition/active types=] to |callbackOrOptions|'s {{StartViewTransitionOptions/types}}.
```

Comment:
These also need to be reset, likely when the finish promise is dispatched or sooner if we're skipping the transition.

I'm not sure how to integrate new functionality like this with algorithms which are in the L1 spec...

Reply:
Why does that matter? That transition is no longer going to be active.

Reply:
Oh yea, my bad.